compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -5355,7 +5367,8 @@ OMR::X86::TreeEvaluator::binaryVectorMaskHelper(TR::InstOpCode opcode,
                                                 TR::Register *lhsReg,
                                                 TR::Register *rhsReg,
                                                 TR::Register *maskReg,
-                                                TR::CodeGenerator *cg)
+                                                TR::CodeGenerator *cg,
+                                                bool maskTypeMismatch)
```

Comment:
Why isn't this called `forceEmulatedMasking` like on the caller side?  To me that name is more descriptive as to what the flag's purpose is.

Reply:
Yes, they should be consistent. However, I think type mismatch is the more appropriate term because that is the problem it intends to solve and on AVX-512 the implementation still uses native masking, just with an extra move instruction.